Controller/Adminhtml/Report/Save.php

```diff
@@ -202,9 +202,17 @@ public function updateVarnishConfiguration()
         $application_id = $this->state->getApplicationId();
         /** @var string $environment_name */
         $environment_name = $this->state->getEnvironmentName();
+        /** @var string $proxy_image*/
```

Comment:
This code block apparently duplicated here and in UpdateVclCommand.php seems like an opportunity to refactor.

Reply:
Done